iroh-net/src/hp/derp/http.rs

```diff
@@ -1,5 +1,5 @@
 mod client;
-mod mesh_clients;
+pub(crate) mod mesh_clients;
```

Comment:
why this change?

Reply:
ahh, that was a mistake, changed it back, ty